### ISSUE TITLE
hostname "localhost" is hardcoded, for example for logging, althrough i use other hostname.

### ISSUE THREAD

**Issue by danilovmy** [denoland/fresh#1833]
```
// fresh/src/server/boot.ts
export async function startFromContext(ctx: ServerContext, opts: StartOptions) {
      opts = { hostname: "0.0.0.0", port: 8080 }
      ...
      const address = colors.cyan(`http://localhost:${params.port}/`);
      ...
}
```

The protocol is also "hardcoded"

**Comment by marvinhagemeister** [comment]
Resolved by #1839

**Comment by king8fisher** [comment]
@marvinhagemeister I was about to jump into this, but you think #1839 addresses this? 
Nevertheless, printing out a server IP address in logs might not be a good idea either. It would have to be conditional on the `dev` flag at least?
